Makefile

```
@@ -75,12 +75,19 @@ integration-tests: $(INSTALL_STAMP)  ##  Run integration tests
 integration-test-fixtures: $(INSTALL_STAMP)  ##  List fixtures in use per integration test
 	MERINO_ENV=testing $(POETRY) run pytest $(INTEGRATION_TEST_DIR) --fixtures-per-test
 
+.PHONY: docker-build
+docker-build:
```

```suggestion
docker-build:  ## Build the docker image for Merino named "app:build"
```
The comment after "##" will be printed out when typing "make help"